src/components/Header/Header.tsx

```diff
@@ -59,13 +62,13 @@ const Header: React.FC<IHeaderProps> = ({ showDetails, movie, changeView }) => {
         <h1>
           <Logo />
         </h1>
-        {showDetails ? (
+        {movie ? (
```

Comment:
why is 'movie' not optional in this component?
if you check its existence, it should be optional.